src/main/java/mate/jdbc/dao/ManufacturerDaoImpl.java

```diff
@@ -0,0 +1,122 @@
+                        PreparedStatement createFormatStatement =
+                                connection.prepareStatement(getByIndexRequest)) {
+            createFormatStatement.setLong(1, id);
+            ResultSet resultSet = createFormatStatement.executeQuery();
+            while (resultSet.next()) {
+                manufacturer = createManufacturer(resultSet);
+            }
+        } catch (SQLException throwables) {
+            throw new DataProcessingException("Unable to return object by ID " + id, throwables);
+        }
+        return Optional.ofNullable(manufacturer);
+    }
+
+    @Override
+    public List<Manufacturer> getAll() {
+        String getAllRequest = "SELECT * FROM manufacturers "
+                + "WHERE is_deleted = false";
+        List<Manufacturer> manufacturerList = new ArrayList<>();
+        try (Connection connection = ConnectionUtil.getConnection();
+                        PreparedStatement createFormatStatement =
+                                connection.prepareStatement(getAllRequest)) {
+            ResultSet resultSet = createFormatStatement.executeQuery();
+            while (resultSet.next()) {
+                Manufacturer manufacturer = createManufacturer(resultSet);
+                manufacturerList.add(manufacturer);
+            }
+        } catch (SQLException throwables) {
+            throw new DataProcessingException("Unable to create list of manufacturers.",
+                    throwables);
+        }
+        return manufacturerList;
+    }
+
+    @Override
+    public Manufacturer update(Manufacturer manufacturer) {
+        String updateRequest = "UPDATE manufacturers "
+                + "SET name = (?), country = (?) WHERE id = (?)";
+        try (Connection connection = ConnectionUtil.getConnection();
+                        PreparedStatement createFormatStatement =
+                                connection.prepareStatement(updateRequest)) {
+            createFormatStatement.setString(1, manufacturer.getName());
+            createFormatStatement.setString(2, manufacturer.getCountry());
+            createFormatStatement.setLong(3, manufacturer.getId());
+            createFormatStatement.executeUpdate();
+        } catch (SQLException throwables) {
+            throw new DataProcessingException("Unable to exectue update with "
+                    + manufacturer, throwables);
+        }
+        return manufacturer;
+    }
+
+    @Override
+    public boolean delete(Long id) {
+        String deleteRequest = "UPDATE manufacturers "
+                + "SET is_deleted = true WHERE id = ?";
+        try (Connection connection = ConnectionUtil.getConnection();
+                        PreparedStatement createFormatStatement =
+                                connection.prepareStatement(deleteRequest)) {
+            createFormatStatement.setLong(1, id);
+            return createFormatStatement.executeUpdate() >= 1;
```

Comment:
[checklist](https://mate-academy.github.io/jv-program-common-mistakes/java-JDBC/jdbc-intro/JDBC-intro_checklist.html)
```suggestion
            return createFormatStatement.executeUpdate() > 0 ;
```